PyPDF2/constants.py

```diff
@@ -302,6 +302,7 @@ class FieldDictionaryAttributes:
     V = "/V"  # text string, optional
     DV = "/DV"  # text string, optional
     AA = "/AA"  # dictionary, optional
+    Opt = "/Opt" # Options, Optional
```

Comment:
This attribute is not defined in Table 8.69 but in Table 8.76. Please use

```suggestion
class CheckboxRadioButtonAttributes:
    """TABLE 8.76 Additional entry specific to check box and radio button fields."""
    Opt = "/Opt" # Options, Optional
```

The main reason for using those constants is to point to the right part of the documentation / adding additional information.

Reply:
where is the table/documentation stored at?